cli/src/commands/content/updateChannelCensorshipStatus.ts

```diff
@@ -0,0 +1,77 @@
+import ContentDirectoryCommandBase from '../../base/ContentDirectoryCommandBase'
+import chalk from 'chalk'
+import ExitCodes from '../../ExitCodes'
+
+export default class UpdateChannelCensorshipStatusCommand extends ContentDirectoryCommandBase {
+  static description = 'Update Channel censorship status (Active/Inactive).'
+  static flags = {
+    context: ContentDirectoryCommandBase.contextFlag,
+  }
+  static args = [
+    {
+      name: 'id',
+      required: false,
+      description: 'ID of the Channel',
+    },
+    {
+      name: 'status',
+      required: false,
+      description: 'New status of the channel (1 - active, 0 - inactive)',
+    },
+    {
+      name: 'rationale',
+      required: false,
+      description: 'rationale',
+    },
+  ]
+
+  async run() {
+    let { context } = this.parse(UpdateChannelCensorshipStatusCommand).flags
+
+    let { id, status, rationale } = this.parse(UpdateChannelCensorshipStatusCommand).args
+
+    if (!context) {
+      context = await this.promptForContext()
+    }
+
+    const currentAccount = await this.getRequiredSelectedAccount()
+    await this.requestAccountDecoding(currentAccount)
+
+    const actor = await this.getActor(context)
+
+    if (id === undefined) {
+      id = await this.promptForChannel()
+    }
+
+    if (status === undefined) {
+      status = await this.simplePrompt({
+        type: 'list',
+        message: 'Select new status',
+        choices: [
+          { name: 'Active', value: true },
+          { name: 'Inactive', value: false },
+        ],
+      })
+    } else {
+      if (status !== '0' && status !== '1') {
+        this.error('Invalid status provided. Use "1" for Active and "0" for Inactive.', {
+          exit: ExitCodes.InvalidInput,
+        })
+      }
+      status = !!parseInt(status)
+    }
+
+    await this.sendAndFollowNamedTx(currentAccount, 'content', 'updateChannelCensorshipStatus', [
```

Comment:
Another reason why this command fails is that there is no `IsCensored` type in `@joystream/types` (after adding new type to the library you need to run a the current node locally and then run `yarn workspace @joystream/types generate:all` to update the api augmentations and a few other auto-generated files)

Reply:
Done https://github.com/Joystream/joystream/pull/2280/commits/8635bea118d08e85f862bc00685b44d3c241db15